src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
+const TOPIC_SEED_CONTAINER = process.env.TOPIC_SEED_CONTAINER;
+
+module.exports = {
+
+  Get: (blobName, id) => {
```

Comment:
Can the name of this function be a bit more descriptive? Also, not sure what `id` is.